docs/client-connectors/databricks-client/install-databricks-connector.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 10
+title: Install Databricks Client Connector
+---
+
+This page explains how to install the Databricks Client Connector.
+
+## Prerequisites
+
+- Databricks Runtime 11.3 LTS(with Apache Spark 3.3.0)
+- Scala 2.12 or later
+- A Macrometa account with write permissions for streams. For more information about permissions, refer to [Permissions](../../account-management/permissions/).
+
+## Install the Macrometa Databricks Client Connector
```

Comment:
I reformatted this procedure quite a bit, please read carefully to make sure it still makes sense.

Reply:
cc: @praminda-macrometa @koshy-thomas

Reply:
Added a comment. other than that, looks ok to me.